src/View/mainpage/DicomView.py

```diff
@@ -53,6 +68,37 @@ def init_view(self):
         background_brush = QtGui.QBrush(
             QtGui.QColor(0, 0, 0), QtCore.Qt.SolidPattern)
         self.view.setBackgroundBrush(background_brush)
+    
+    def wheelEvent(self, event: QtGui.QWheelEvent):
+        modifiers = event.modifiers()
+        if modifiers == QtCore.Qt.ShiftModifier:
+            super().wheelEvent(event)
+        elif modifiers == QtCore.Qt.AltModifier:
+            super().wheelEvent(event)
+        elif modifiers == QtCore.Qt.ControlModifier:
+            angle = event.angleDelta() / 8
```

Comment:
8 and 15 are "magic" numbers.
Are these the recommended values in Qt example code?
Is there a URL reference that could be put in comments in this function that would point someone to the reason why eight and fifteen are appropriate values.
I found:
https://doc.qt.io/qt-6/qwheelevent.html#angleDelta
and it seems like the appropriate value might be different with a high resolution mouse.  A RadOnc might have a high resolution mouse...

This is probably OK for now, but it might need to be adjustable in some way in the future, e.g. by a configuration item by the user, or by somehow querying the system  to see if more information is available.